packages/touchpoint-ui/src/App.tsx

```diff
@@ -39,7 +39,19 @@ export interface Props {
   config: Config;
   windowSize?: WindowSize;
   colorMode?: ColorMode;
+  /**
+   * Logo URL
+   * @deprecated - use brandIcon and launchIcon intead
+   */
   logoUrl?: LogoUrl;
+  /**
+   * Icon used to display the brand in the chat header
```

Comment:
```suggestion
   * URL to icon used to display the brand in the chat header
```

Reply:
Updated both places, thanks for the catch!